src/main/java/minegame159/meteorclient/gui/screens/topbar/TopBarConfig.java

```diff
@@ -61,6 +61,15 @@ protected void initWidgets() {
                 .build()
         );
 
+        sgGeneral.add(new BoolSetting.Builder()
+                .name("send-data-to-api")
+                .description("If checked, your UUID will be send to Meteor's servers.")
+                .defaultValue(Config.get().sendDataToApi)
```

Comment:
Also this should be true, not the value that it is currently set to

Reply:
ah, ok